http-common/src/connector.rs

```diff
@@ -27,8 +27,15 @@ pub enum AsyncStream {
 #[cfg(feature = "tokio1")]
 #[derive(Debug)]
 pub enum Incoming {
-    Tcp(tokio::net::TcpListener),
-    Unix(tokio::net::UnixListener),
+    Tcp {
+        listener: tokio::net::TcpListener,
+    },
+    Unix {
+        listener: tokio::net::UnixListener,
+        user_state: futures_util::lock::Mutex<
```

Comment:
This doesn't need to be an async mutex. Just use `std::sync::Mutex`.

Reply:
Fixed